convert-hf-to-gguf.py

```diff
@@ -177,7 +177,7 @@ def match_model_tensor_name(self, name: str, key: gguf.MODEL_TENSOR, bid: int |
                 return False
         return name == (key_name + suffix)
 
-    def map_tensor_name(self, name: str, try_suffixes: Sequence[str] = (".weight", ".bias")) -> str:
+    def map_tensor_name(self, name: str, try_suffixes: Sequence[str] = (".weight", ".bias", ".beta", ".gamma")) -> str:
```

Comment:
These are the global defaults of name suffixes to try which I think in this case should instead be overwritten only for Bert.

Since all the Bert-like models use the `BertModel` class as their base class, you can actually use

```python3
    def map_tensor_name(self, name: str, try_suffixes: Sequence[str] = (".weight", ".bias", ".beta", ".gamma")) -> str:
        return super().map_tensor_name(name, try_suffixes)
```

in the `BertModel` class to change the defaults only for these models.

Reply:
> the .beta and .gamma weights are not currently handled in the llama.cpp file
How much effort do you think it would take to implement this?

Reply:
> How much effort do you think it would take to implement this?

Actually, from <https://www.tensorflow.org/api_docs/python/tf/keras/layers/LayerNormalization>, maybe it could be as simple as renaming `.gamma` to `.weight`, and `.beta` to `.bias`.

Reply:
```
name = name.replace("gamma", "weight")
name = name.replace("beta", "bias")
```

So I went ahead and tried that. Looks like the pooling type is not getting set correctly.

```
GGML_ASSERT: llama.cpp:12243: lctx.embd != nullptr
Could not attach to process.  If your uid matches the uid of the target
process, check the setting of /proc/sys/kernel/yama/ptrace_scope, or try
again as the root user.  For more details, see /etc/sysctl.d/10-ptrace.conf
ptrace: Operation not permitted.
No stack.
The program is not being run.
Aborted (core dumped)
```

Reply:
> `GGML_ASSERT: llama.cpp:12243: lctx.embd != nullptr`

In your latest commit, this assertion is on line 12317. I can't help with debugging this if I don't have access to the source you're using.

As a temporary workaround, you can use the `--pooling` option of the `embedding` example if you want to set it.
For example:

```console
$ ./build/bin/embedding -m ./models/your-bert-model.gguf --pooling mean -p "Hello world"
```